gengokit/httptransport/templates/server.go

```diff
@@ -179,12 +179,33 @@ func MakeHTTPHandler(ctx context.Context, endpoints Endpoints, logger log.Logger
 	return m
 }
 
+// ErrorEncoder writes the error to the ResponseWriter, by default a content
+// type of application/json, a body of json with key "error" and the value
+// error.Error(), and a status code of 500. If the error implements Headerer,
+// the provided headers will be applied to the response. If the error
+// implements json.Marshaler, and the marshaling succeeds, the JSON encoded
+// form of the error will be used. If the error implements StatusCoder, the
+// provided StatusCode will be used instead of 500.
 func errorEncoder(_ context.Context, err error, w http.ResponseWriter) {
+	contentType := "application/json; charset=utf-8"
```

Comment:
This should probably be a constant. Although it's only used once so someone could probably make the argument that it should just be used inline, but I don't feel strongly either way here.